src/encoded/static/components/filegallery.js

```diff
@@ -151,7 +151,13 @@ export class FileTable extends React.Component {
             // Extract four kinds of file arrays
             const files = _(datasetFiles).groupBy((file) => {
                 if (file.output_category === 'raw data') {
-                    return file.output_type === 'reads' ? 'raw' : 'rawArray';
+                    if (file.output_type === 'reads') {
```

Comment:
What an if- else if be a tad better? Isolated if-statements imply the if-statements are not related nor operating on the same variable.

Reply:
With embedded returns, that wouldn’t be allowed under [AirBnB](https://github.com/airbnb/javascript#arrays--callback-return).